server/coordinator.go

```diff
@@ -521,6 +553,16 @@ func (c *coordinator) sendScheduleCommand(region *core.RegionInfo, step schedule
 			},
 		}
 		c.hbStreams.sendMsg(region, cmd)
+	case schedule.MergeRegion:
+		if s.IsPassive {
```

Comment:
what is passive?

Reply:
you can see the explanation at where it defines